test/OrchardCore.Commerce.Tests.UI/Tests/OrderTests/OrderSuccessTests.cs

```diff
@@ -49,11 +49,11 @@ public OrderSuccessTests(ITestOutputHelper testOutputHelper)
                 await AddProductToCartAsync(context, TestPriceVariantProduct);
 
                 await context.ClickAndFillInWithRetriesAsync(QuantityFieldBy(1), "2");
-                await context.ClickAndFillInWithRetriesAsync(QuantityFieldBy(2), "3");
+                await context.ClickAndFillInWithRetriesAsync(QuantityFieldBy(2), "2");
```

Comment:
This was set to an invalid value before, but the logic to check for its validity wasn't in place yet.